queries.sql

```diff
@@ -1,3 +1,28 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
\ No newline at end of file
+SELECT * FROM animals WHERE name LIKE '%mon' ;
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-01' ;
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3 ;
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu' ;
+SELECT name, escape attempts FROM animals WHERE weight_kg > 10.5 ;
+SELECT * FROM animals WHERE neutered = true ;
+SELECT * FROM animals WHERE name NOT IN ('Gabumon') ;
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3 ;
+BEGIN ;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
```

Comment:
- [ ] Kindly verify that the change was made using the `SELECT` query before and after you rollback :pray: